src/main/java/org/zaproxy/zest/core/v1/ZestClientElementScroll.java

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+package org.zaproxy.zest.core.v1;
+
+import org.openqa.selenium.JavascriptExecutor;
+
+/**
+ * Scroll in the current frame by (x,y) co-ordinates
+ *
+ * @author aryan
+ */
+public class ZestClientElementScroll extends ZestClientElement {
+
+    private int Xvalue = 0;
+    private int Yvalue = 0;
```

Comment:
- Variables should start with lower case, `x` and `y` seem fine.
 - The initialisation to 0 is redundant.